Zend/zend_portability.h

```diff
@@ -435,7 +435,7 @@ char *alloca();
 #ifdef ZEND_WIN32
 #define ZEND_SECURE_ZERO(var, size) RtlSecureZeroMemory((var), (size))
 #else
-#define ZEND_SECURE_ZERO(var, size) memset((var), 0, (size))
+#define ZEND_SECURE_ZERO(var, size) explicit_bzero((var), (size))
```

Comment:
Just wondering, do we need a fallback here in case explicit_bzero is not available on Unix? Like:

...
# ifdef HAVE_EXPLICIT_BZERO
#  define ZEND_SECURE_ZERO(var, size) explicit_bzero((var), (size))
# else
#  define ZEND_SECURE_ZERO(var, size) memset((var), 0, (size))
# endif
...